From c94fcb2b120b468bb7758d8d85fbc65b925ca6f5 Mon Sep 17 00:00:00 2001
From: Mike Gerdts <mike.gerdts@joyent.com>
Date: Wed, 9 May 2018 12:18:23 +0000
Subject: [PATCH] OS-6802 zoneadmd child process pollutes environment (fix
 DEBUG) Reviewed by: John Levon <john.levon@joyent.com> Reviewed by: Dan
 McDonald <danmcd@joyent.com> Approved by: Dan McDonald <danmcd@joyent.com>

---
 usr/src/cmd/zoneadmd/zoneadmd.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/usr/src/cmd/zoneadmd/zoneadmd.c b/usr/src/cmd/zoneadmd/zoneadmd.c
index 4599453d86..21c6460eeb 100644
--- a/usr/src/cmd/zoneadmd/zoneadmd.c
+++ b/usr/src/cmd/zoneadmd/zoneadmd.c
@@ -1039,7 +1039,6 @@ do_subproc(zlog_t *zlogp, char *cmdbuf, char **retstr, boolean_t debug)
 
 	if ((child = fork()) == 0) {
 		int in;
-		struct sigaction sa = { 0 };
 
 		/*
 		 * SIGINT is currently ignored.  It probably shouldn't be so
-- 
2.21.0

